examples/utils/get-tinylicious-container/src/getTinyliciousContainer.ts

```diff
@@ -25,12 +25,14 @@ import { getContainer } from "./getContainer";
  * URL shape, so it expects the request url to have this format (as opposed to a more traditional URL):
  * documentId/containerRelativePathing
  */
-class InsecureTinyliciousUrlResolver implements IUrlResolver {
+export class InsecureTinyliciousUrlResolver implements IUrlResolver {
     public async resolve(request: IRequest): Promise<IResolvedUrl> {
-        const documentId = request.url.split("/")[0];
+        const url = request.url;
+        const documentId = url.split("/")[0];
         const encodedDocId = encodeURIComponent(documentId);
+        url.replace(documentId,encodedDocId);
 
-        const documentUrl = `fluid://localhost:3000/tinylicious/${encodedDocId}`;
+        const documentUrl = `fluid://localhost:3000/tinylicious/${url}`;
```

Comment:
nit: Consider getting both "docId" and "documentRelativePath" and appending it to the "documentUrl" (similar to how its done in InsecureUrlResolver and odspDriverUrlResolver). It makes it explicit and the code clearer.